[0m18:02:11.305804 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcf31d93b60>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcf2ff93440>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcf2fe2c200>]}


============================== 18:02:11.310512 | bd306aef-6122-435a-8a2b-8b643a985c58 ==============================
[0m18:02:11.310512 [info ] [MainThread]: Running with dbt=1.9.4
[0m18:02:11.311048 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'debug': 'False', 'profiles_dir': '/home/bedo7/.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'static_parser': 'True', 'invocation_command': 'dbt init my_dbt_project', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m18:02:11.329092 [debug] [MainThread]: Starter project path: /home/bedo7/learning/dbt/venv/lib/python3.12/site-packages/dbt/include/starter_project
[0m18:02:11.338389 [info ] [MainThread]: 
Your new dbt project "my_dbt_project" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m18:02:11.338872 [info ] [MainThread]: Setting up your profile.
[0m18:18:21.887564 [info ] [MainThread]: Profile my_dbt_project written to /home/bedo7/.dbt/profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m18:18:21.942265 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 999.7328, "process_in_blocks": "54856", "process_kernel_time": 0.122481, "process_mem_max_rss": "100320", "process_out_blocks": "80", "process_user_time": 1.535299}
[0m18:18:21.946360 [debug] [MainThread]: Command `dbt init` succeeded at 18:18:21.945348 after 999.74 seconds
[0m18:18:21.958665 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcf31d93b60>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcf2f9b4140>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcf2faceb10>]}
[0m18:18:21.971945 [debug] [MainThread]: Flushing usage events
[0m18:18:22.967983 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m18:18:33.777613 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fd4bb303290>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fd4bacffec0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fd4bb301f70>]}


============================== 18:18:33.783798 | 84241607-9d3c-44ee-abaf-9b0df5b0e106 ==============================
[0m18:18:33.783798 [info ] [MainThread]: Running with dbt=1.9.4
[0m18:18:33.784414 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/home/bedo7/.dbt', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'debug': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'invocation_command': 'dbt debug', 'static_parser': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m18:18:33.800804 [info ] [MainThread]: dbt version: 1.9.4
[0m18:18:33.801323 [info ] [MainThread]: python version: 3.12.3
[0m18:18:33.801672 [info ] [MainThread]: python path: /home/bedo7/learning/dbt/venv/bin/python
[0m18:18:33.801997 [info ] [MainThread]: os info: Linux-5.15.167.4-microsoft-standard-WSL2-x86_64-with-glibc2.39
[0m18:18:33.872856 [info ] [MainThread]: Using profiles dir at /home/bedo7/.dbt
[0m18:18:33.873357 [info ] [MainThread]: Using profiles.yml file at /home/bedo7/.dbt/profiles.yml
[0m18:18:33.873704 [info ] [MainThread]: Using dbt_project.yml file at /home/bedo7/learning/dbt/dbt_project.yml
[0m18:18:33.874083 [error] [MainThread]: Encountered an error:
Internal Error
  Profile should not be None if loading profile completed
[0m18:18:33.874914 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_success": false, "command_wall_clock_time": 0.15798536, "process_in_blocks": "56", "process_kernel_time": 0.450899, "process_mem_max_rss": "100052", "process_out_blocks": "8", "process_user_time": 1.303686}
[0m18:18:33.875349 [debug] [MainThread]: Command `dbt debug` failed at 18:18:33.875255 after 0.16 seconds
[0m18:18:33.875680 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fd4baa2c620>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fd4ba7900e0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fd4baa65730>]}
[0m18:18:33.876033 [debug] [MainThread]: Flushing usage events
[0m18:18:34.827247 [debug] [MainThread]: An error was encountered while trying to flush usage events
